provision/initramfs/init

```diff
@@ -480,9 +480,8 @@ for i in /etc/warewulf/init/*; do
     fi
 done
 
-
+mkdir -p /var/log/warewulf/provision
```

Comment:
Can you move this up to just above the:
```
msg_blue "Now Booting Warewulf...\n"
```
line? Just make the directory at the start of the init file.

Reply:
OK, changed.

Reply:
Thanks.